### PR TITLE
Add example for RabbitMQ stream queues

### PR DESCRIPTION
**Pull request by Gsantomaggio**
It shows how to use the stream queue with the AMQP client.

See: 
- https://www.rabbitmq.com/stream.html
- https://blog.rabbitmq.com/posts/2021/07/rabbitmq-streams-overview

### CODE REVIEW

**Comment by carlhoerberg** [comment]
The stream plugin is not required for this example, only Rabbitmq 3.9 which has stream queue support built-in, without the need for a plugin. The plugin only adds custom protocol support.

**Comment by Gsantomaggio** [comment]
Ops sorry you are right! I changed the description! 
thank you

File: examples/stream_queues/receive_stream.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+
+var amqp = require('amqplib');
```
Comment (kibertoad) [inline examples/stream_queues/receive_stream.js:3]: can you please rewrite to use modern style with `const`/`let` and `async/await`?

File: examples/stream_queues/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+The [stream queues](https://www.rabbitmq.com/streams.html) are available starting from RabbitMQ 3.9
+
+These examples show how to use stream queues with the lib.
+
+Send a message to a stream queue
+```
+./send_stream.js   
+```
+
+Reveice all the messages from stream queue:
```
Comment (kibertoad) [inline examples/stream_queues/README.md:12]:
```suggestion
Receive all the messages from stream queue:
```

**Review by kibertoad — APPROVED** [review]

**Pull request merged**